examples/customer-api/server.ts

```diff
@@ -102,7 +103,7 @@ export default {
  * Feel free to customize it to your needs, add helper methods, or
  * swap out the cookie-based implementation with something else!
  */
-export class HydrogenSession {
+export class HydrogenSession implements DefaultHydrogenSession {
```

Comment:
Somewhat nit-picky, but I think that `DefaultHydrogenSession` doesn't really make sense as an identifier name. It is the interface that the developer must conform to. So it isn't really a "default". It is what they need to implement to ensure compatibility. IMO `export class Session implements HydrogenSession` makes more sense. But that means a bit more to modify.

Reply:
Ahh, I see Fran above had a similar comment. Whether it is `Session` or `CookieSession`, I just don't like calling the type `DefaultHydrogenSession` in our examples.